jaxrs/WFLY-14559_Add_six_new_RESTEasy_context_parameters.adoc

```diff
@@ -0,0 +1,102 @@
+= WFLY-14559 Add six new RESTEasy context parameters
+:author:            Ron Sigal
+:email:             rsigal@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse
+
+== Overview
+
+Since the RESTEasy context parameters were first exposed in the wildfly/jaxrs management model
+(https://github.com/wildfly/wildfly-proposals/blob/main/jaxrs/WFLY-12298_Change_RESTEASY_settings.adoc),
+several new parameters have been introduced, of which the following should be exposed to the user:
+
+ * resteasy.match.cache.enabled:                       [RESTEASY-2646]
+ * resteasy.match.cache.size:                          [RESTEASY-2646]
+ * resteasy.original.webapplicationexception.behavior: [RESTEASY-2782]
+ * resteasy.patch.filter.disabled:                     [RESTEASY-2280]
+ * resteasy.patch.filter.legacy:                       [RESTEASY-2849]
+ * resteasy.proxy.implement.all.interfaces:            [RESTEASY-2866]
+
+The wildfly/jaxrs module should be extended to expose these new parameters.
+
+[Note. RESTEasy uses '.' as a separator in parameter names. WildFly/jaxrs uses '-'. 
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}/EAP7-1655[EAP7-1655]
+* {issue-base-url}/WFLY-14559[WFLY-14559]
+
+=== Related Issues:
+
+* {issue-base-url}/RESTEASY-2280[RESTEASY-2280]
+* {issue-base-url}/RESTEASY-2646[RESTEASY-2646]
+* {issue-base-url}/RESTEASY-2782[RESTEASY-2782]
+* {issue-base-url}/RESTEASY-2849[RESTEASY-2849]
+* {issue-base-url}/RESTEASY-2866[RESTEASY-2866]
+
+=== Dev Contacts:
+
+* mailto:rsigal@redhat.com[Ron Sigal]
+
+=== QE Contacts:
+
+Jan Blizňák
+
+=== Testing By
+
+Engineering
+
+=== Affected Projects or Components:
+
+* WildFly
+* RESTEasy
+
+== Requirements
+
+* Some simple modifications to the wildfly/jaxrs module will be necessary to add these new parameters.
+* This affects the web interface.
+
+=== Test Plan
+
+The proper behavior of the existing parameters is already tested in RESTEasy, as follows:
+
+ * resteasy.match.cache.enabled, resteasy.match.cache.size:
+ 
+   resteasy-core/src/test/java/org/jboss/resteasy/core/registry/RootNodeCacheSizeTest.java   
+    
+ * resteasy.original.webapplicationexception.behavior:
+ 
+   testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/client/exception/ClientWebApplicationExceptionMicroProfileProxyTest.java
```

Comment:
Should these be links? I'm not convinced they should be, but it seems like we could easily enough make them links to the source.

Reply:
Done, except for the WildFly tests. I guess I could add those after WildFly is released.